### PR TITLE
add CLI shell documentation

### PR DESCRIPTION
**Pull request by jul-dan**

### CODE REVIEW

File: website/docs/using-qovery/interface/cli.md.erb [hunk 1]
```diff
@@ -199,6 +199,77 @@ Keep in mind these limitations when using this feature:
 </ul>
 </Alert>
 
+<Alert type="info">
```
Comment (acarranoqovery) [inline website/docs/using-qovery/interface/cli.md.erb:202]: in the alert above there's still the phrase "if your application is configured to run with more than one instance, you will be able to connect only to the oldest POD running one of your instances"

File: website/docs/using-qovery/interface/cli.md.erb [hunk 2]
```diff
@@ -199,6 +199,77 @@ Keep in mind these limitations when using this feature:
...
+The width of the terminal is limited to 80 characters. But you can resize it once you are inside the application with one of these commands:
```
Comment (acarranoqovery) [inline website/docs/using-qovery/interface/cli.md.erb:203]: shell is force closed after [1 hour, 10Mb transmitted]; <-- still true the 10Mb limitation?

**Review by acarranoqovery — CHANGES_REQUESTED** [review]

**Review by acarranoqovery — APPROVED** [review]
ok, waiting for the feedback from Romain

File: website/docs/using-qovery/interface/cli.md.erb (hunk 2 above)
Comment (jul-dan) [reply website/docs/using-qovery/interface/cli.md.erb:202]: It is 1GB now, I have updated

**Pull request merged**